### PR TITLE
updated names-generator.go for alphabetization

### PR DESCRIPTION
**Pull request by jk-vb**
re ordered some entries so they are in proper alphabetical order

**- What I did**
alphabetize some entries  
**- How I did it**
github editor
**- How to verify it**

**- Description for the changelog**

Updated entries in names-generator.go for proper alphabetization

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by thaJeztah** [comment]
Thanks for contributing; looks like your commit is missing a DCO sign-off line, which causes CI to fail; see https://github.com/moby/moby/blob/master/CONTRIBUTING.md#sign-your-work

Unfortunately, it's not possible to amend commits from GitHub's UI, so you will have to do this from a command-line to amend  the commit message; the DCO bot provides some instructions on how to do this; https://github.com/moby/moby/pull/42614/checks?check_run_id=3032256187, but let me know if you need a hand.

**Comment by jk-vb** [comment]
> Thanks for contributing; looks like your commit is missing a DCO sign-off line, which causes CI to fail; see https://github.com/moby/moby/blob/master/CONTRIBUTING.md#sign-your-work
> 
> Unfortunately, it's not possible to amend commits from GitHub's UI, so you will have to do this from a command-line to amend the commit message; the DCO bot provides some instructions on how to do this; https://github.com/moby/moby/pull/42614/checks?check_run_id=3032256187, but let me know if you need a hand.

I think I signed it correctly now

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -248,19 +248,19 @@ var (
 
 		// Asima Chatterjee was an Indian organic chemist noted for her research on vinca alkaloids, development of drugs for treatment of epilepsy and malaria - https://en.wikipedia.org/wiki/Asima_Chatterjee
 		"chatterjee",
-
+
```
Comment (thaJeztah) [inline pkg/namesgenerator/names-generator.go:251]: I think `gofmt` doesn't like the extra empty line;

```
pkg/namesgenerator/names-generator.go:125: File is not `goimports`-ed (goimports)
```

Let me run `gofmt` and push

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Review by tianon — APPROVED** [review]

**Comment by tianon** [comment]
(Windows) CI failure is unrelated: :joy:

```console
[2021-07-13T23:36:46.512Z] === FAIL: github.com/docker/docker/integration/build TestBuildWCOWSandboxSize (680.94s)

[2021-07-13T23:36:46.512Z]     build_test.go:572: assertion failed: value "{\"stream\":\"Step 1/8 : FROM busybox AS intermediate\"}\r\n{\"stream\":\"\\n\"}\r\n{\"stream\":\" ---\\u003e 4d052f67f873\\n\"}\r\n{\"stream\":\"Step 2/8 : WORKDIR C:\\\\\\\\stuff\"}\r\n{\"stream\":\"\\n\"}\r\n{\"stream\":\" ---\\u003e Running in aab7be56a31c\\n\"}\r\n{\"stream\":\"Removing intermediate container aab7be56a31c\\n\"}\r\n{\"stream\":\" ---\\u003e 651d3a0d4aa3\\n\"}\r\n{\"stream\":\"Step 3/8 : RUN fsutil file createnew C:\\\\\\\\stuff\\\\\\\\bigfile_0.txt 22548578304 \\u0026\\u0026 del bigfile_0.txt\"}\r\n{\"stream\":\"\\n\"}\r\n{\"stream\":\" ---\\u003e Running in 24bee670406d\\n\"}\r\n{\"stream\":\"File C:\\\\stuff\\\\bigfile_0.txt is created\\r\\n\"}\r\n{\"stream\":\"Removing intermediate container 24bee670406d\\n\"}\r\n{\"stream\":\" ---\\u003e 006bbc4a532c\\n\"}\r\n{\"stream\":\"Step 4/8 : RUN fsutil file createnew C:\\\\\\\\stuff\\\\\\\\bigfile_1.txt 7516192768\"}\r\n{\"stream\":\"\\n\"}\r\n{\"stream\":\" ---\\u003e Running in ac6c188a766f\\n\"}\r\n{\"stream\":\"File C:\\\\stuff\\\\bigfile_1.txt is created\\r\\n\"}\r\n{\"stream\":\"Removing intermediate container ac6c188a766f\\n\"}\r\n{\"stream\":\" ---\\u003e 79b12e1a06fc\\n\"}\r\n{\"stream\":\"Step 5/8 : RUN fsutil file createnew C:\\\\\\\\stuff\\\\\\\\bigfile_2.txt 7516192768\"}\r\n{\"stream\":\"\\n\"}\r\n{\"stream\":\" ---\\u003e Running in 90e54b278392\\n\"}\r\n{\"stream\":\"File C:\\\\stuff\\\\bigfile_2.txt is created\\r\\n\"}\r\n{\"stream\":\"Removing intermediate container 90e54b278392\\n\"}\r\n{\"stream\":\" ---\\u003e 7f00fb4426c9\\n\"}\r\n{\"stream\":\"Step 6/8 : RUN fsutil file createnew C:\\\\\\\\stuff\\\\\\\\bigfile_3.txt 7516192768\"}\r\n{\"stream\":\"\\n\"}\r\n{\"stream\":\" ---\\u003e Running in ef7f928306ae\\n\"}\r\n{\"stream\":\"File C:\\\\stuff\\\\bigfile_3.txt is created\\r\\n\"}\r\n{\"stream\":\"Removing intermediate container ef7f928306ae\\n\"}\r\n{\"stream\":\" ---\\u003e 66d01a416872\\n\"}\r\n{\"aux\":{\"ID\":\"sha256:66d01a4168723129d6037ca7bf30e91c2aded1967adf1964702d4107a058da0e\"}}\r\n{\"stream\":\"Step 7/8 : FROM busybox\"}\r\n{\"stream\":\"\\n\"}\r\n{\"stream\":\" ---\\u003e 4d052f67f873\\n\"}\r\n{\"stream\":\"Step 8/8 : COPY --from=intermediate C:\\\\\\\\stuff C:\\\\\\\\stuff\"}\r\n{\"stream\":\"\\n\"}\r\n{\"errorDetail\":{\"message\":\"re-exec error: exit status 1: output: hcsshim::ImportLayer - failed failed in Win32: The system cannot find the path specified. (0x3)\"},\"error\":\"re-exec error: exit status 1: output: hcsshim::ImportLayer - failed failed in Win32: The system cannot find the path specified. (0x3)\"}\r\n" does not match regexp "Successfully built|re-exec error: exit status 1: output: write.*daemon\\\\\\\\tmp\\\\\\\\hcs.*bigfile_[1-3].txt: There is not enough space on the disk."
```

(`There is not enough space on the disk.` - @StefanScherer I believe that might be your domain? :pray: :innocent:)

**Pull request merged**